app/models/service.rb

```diff
@@ -14,8 +14,11 @@ class Service < ApplicationRecord
   scope :at_date, ->(date) { where(arel_table[:beginning].lteq(date)).where(arel_table[:ending].gteq(date)) }
   scope :at_year, ->(year) { overlapping_date_range(Date.new(year), Date.new(year).at_end_of_year) }
   scope :active, -> { where('beginning <= ?', Time.zone.now).where('ending >= ?', Time.zone.now) }
-  scope :agreement, -> { where(civil_servant_agreed: false).or(where(organization_agreed: false)) }
-  scope :definitive, -> { where(civil_servant_agreed: true).where(organization_agreed: true) }
+  scope :civil_servant_agreement_pending, -> { where(civil_servant_agreed: nil) }
+  scope :organization_agreement_pending, -> { where(organization_agreed: nil) }
+  scope :agreement, -> { civil_servant_agreement_pending.or(organization_agreement_pending) }
+  scope :declined, -> { where(organization_agreed: false).or(civil_servant_agreed: false) }
```

Comment:
Does that scope make sense? Do you even use it?

Reply:
Use? Nope.
Make sense? Not sure, but I'll remove I can always readd it in future if we need it